Review-11-Partial-Differential-Equations.ipynb

```diff
@@ -41,7 +41,7 @@
    "source": [
     "> 11.3. True or false: The Gauss-Seidel method is a special case of SOR (successive over-relaxation) for solving a system of linear equations.\n",
     "\n",
-    "False. Gauss-Seidel is a special case of the class of methods referred to as **stationary methods**.  These methods use the technique of splitting to rewrite the matrix $A$ in the system $Ax = b$."
+    "False. Gauss-Seidel is a special case of the class of methods referred to as **stationary methods**.  These methods use the technique of splitting to rewrite the matrix $A$ in the system $Ax = b$. However setting the relaxation parameter value to one in SOR results is equivalent to the Gauss-Seidel method."
```

Comment:
Good point, at the time I wrote the answer it didn't occur to me that SOR uses weighted average of current and previous Gauss-Seidel iterates.